pages/linux/xxd.md

```diff
@@ -0,0 +1,24 @@
+# xxd
+
+> Make a hexdump or do the reverse.
+> More information: <https://packages.debian.org/buster/xxd>.
+
+- Display Hex representation of a file:
```

Comment:
Perhaps we shall adjust this to make the casing concise.
```suggestion
- Display hex representation of a file:
```

Reply:
I agree it certainly should consistent overall!